community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,123 @@
+
+ - **Be empathetic**. Realize that every post, event, project, or program is created by human beings that are likely trying their best, and aren't perfect. Try your best to understand perspectives that differ from your own, and to ask for clarification when you don't understand how someone feels.
+ - **Be respectful**: Every person deserves to be treated with kindness and respect. If we assume good intentions, we can work together to resolve conflict. We may all experience some frustration from time to time, but we do not allow frustration to turn into a personal attack. A community where
+people feel uncomfortable or threatened is not a productive one.
+ - **Be welcoming**: When an interaction or community is new, it can be intimidating. We aspire to be welcoming, friendly, and patient to both new and current members, and remember that everyone is a constantly growing and learning at all points in our lives.
+ - **Be vulnerable**. If you feel the compulsion to act in a disrespectful or otherwise inappropriate manner, you should practice self-compassion, and both step away from the current interaction, and seek others for support. It can be helpful to talk to someone about what is troubling you, and discuss how you can approach a situation with respect, and without conflict. 
+ - **Be collaborative**: The success and spirit of SourceCred depends on the participation of many people. Effective and friendly collaboration enables everyone to better achieve their goals.  
+ - **Be aware**: We are careful and respectful in our communication, and we take responsibility for our own speech. Be aware of how what you say or do might impact others. Be kind to others. Do not insult or put down other members of the community.
+
+<br>
+
+## Unacceptable Behavior
+
+We, the members of the SourceCred community, are committed to making participation in this community a harassment-free experience.
+
+We will not accept harassment or other exclusionary behaviors, such as:
+ * The use of sexualized language or imagery  
+ * Excessive profanity (please avoid curse words; people differ greatly in their sensitivity to swearing)  
+ * Posting sexually explicit or violent material
+ * Violent or intimidating threats or language directed against another person or group
+ * Inappropriate physical contact and/or unwelcome sexual attention or sexual comments
+ * Sexist, racist, or otherwise discriminatory jokes and language
+ * Trolling or insulting and derogatory comments
+ * Written or verbal comments which have the effect of excluding people on the basis of membership in a specific group, including level
+of experience, gender, gender identity and expression, sexual orientation, disability, neurotype, personal appearance, body size, race,
+ethnicity, age, religion, or nationality
+ * Public or private harassment
```

Comment:
okay, let me give a shot at simplifying these points, and then it would be helpful after that for specific feedback.

Reply:
Had a look and they do look better this way.

**Disclosure of personal information of another person without explicit permission**
This looks great :+1:

**Harassment of another individual or group**
Happy, but realized may be duplicate. As above the bullets it says:
"We will not accept harassment or other exclusionary behaviors, such as:"
Small preference to leave it in, but happy either way.

**Discriminatory Written or verbal comments**
Giving this another look I noticed that definitely matches one above it.
"Sexist, racist, or otherwise discriminatory jokes and language"

In the interest of specific feedback, I would suggest letting the existing:
"Sexist, racist, or otherwise discriminatory jokes and language"
Be the case against discrimination. And take the other exclusionary aspects that may be non-obvious just by disallowing discrimination.

Suggesting:
"Unwarranted exclusionary comments, such as based on experience level, disability or appearance."

I don't know if unwarranted is the best word here. But I'm thinking that, a thoughtful argument a role requires a certain experience level, would be an "exclusive comment" but is not what I mean. Unacceptable comments like "we don't need [..unwarranted..] in the community call" is what I'm thinking about.

---

I notice these were removed in the change, so I'll add thoughts on those:

**We don't accept: Continuing to initiate interaction (such as photography, recording, messaging, or conversation) with someone after being asked to stop**
"Continuing to initiate interaction ... with someone" is a little uncommon maybe, but it makes sense with the examples.

I've tried rephrasing and came up with these:
_Continuing behavior which makes someone uncomfortable, after being asked to stop._
_Continued interaction after being asked to stop. Such as recording, messaging or conversation._
_Disrespecting someones boundary after being asked to stop. Such as recording, messaging or conversation._

**We don't accept: Sustained disruption of talks, events, or communications, such as heckling of a speaker**
I think this was good :smiley:. Would like to keep this in.

---

Other than that I think they look good.

Reply:
okay pushing an update soon!

Reply:
LGTM! :+1:
Feel free to resolve this thread.